opentreemap/treemap/util.py

```diff
@@ -231,3 +231,10 @@ def make_udf_name_from_key(key):
 
 def make_udf_lookup_from_key(key):
     return 'udfs__{}'.format(key)
+
+
+def num_format(num):
+    if isinstance(num, float):
+        # Allow for up to 10 digits of precision, but strip trailing 0s
+        return '{0:.10f}'.format(num).rstrip('0')
```

Comment:
For any "whole number" value like `123.0` this `rstrip` is leaving a trailing `.`

<img width="533" alt="screen shot 2017-09-06 at 9 55 21 am" src="https://user-images.githubusercontent.com/17363/30124353-bd3e8c80-92e9-11e7-8b35-71f161c852c1.png">
 



<img width="737" alt="screen shot 2017-09-06 at 10 00 55 am" src="https://user-images.githubusercontent.com/17363/30124549-5be49050-92ea-11e7-93f1-3a25655e5643.png">

Reply:
Fixed in https://github.com/OpenTreeMap/otm-core/pull/3208/commits/7bdee5f995cedd6ee22b6f523e475697726045da